README.md

```diff
@@ -59,22 +60,13 @@ mkdir -p src
 vcs import src < autoware.proj.repos
 ```
 
-2. Run the setup script
+3. Run the setup script (this step will install CUDA, cuDNN 7, osqp and TensorRT 7)

```

Comment:
I remember (and maybe its changed since) but the script also does something with ROS? Is this still true?

Reply:
See comment above (https://github.com/tier4/AutowareArchitectureProposal.proj/pull/188/files#r605390771)